src/app_service/service/chat/dto/chat.nim

```diff
@@ -35,10 +35,16 @@ type
     large*: string
     banner*: string
 
+type RevealedAccount* = object
+  address*: string
+  chainIds*: seq[int]
```

Comment:
Do we really need `chainIDs` here?
Remember that those are currently only needed in status-go to check funds on given chains.
The airdrop address in itself is the same across chains, and I don't think we decide here on what chain the airdrop happens, or am I missing something?

Reply:
I wasn't sure. I thought maybe on the frontend we might need to filter by chainID? Like if the owner only has funds in Optimism, it can't really airdrop on Arbitrum? I could be wrong though

Reply:
So these data are primarily for: on which chains does this member have funds to satisfy permission to join or view/post on a channel.

I think aidropping is rather decoupled from that. You could airdrop to members that don't have any funds anywhere.
Anyways, it surely doesn't hurt to have them here, but I don't think they are needed atm.